gradio/blocks.py

```diff
@@ -2481,7 +2492,7 @@ def get_api_info(self):
                 else:
                     skip_endpoint = True  # if component not found, skip endpoint
                     break
-                type = component["type"]
+                type = component["props"].get("name")
```

Comment:
Isn't `name` always present? I'm thinking we can remove the `get`. I'm not sure it makes sense anyways because as-is the call to `capitalize` below will fail if `type` is None.

Reply:
Sure yep